src/components/common/Notification.tsx

```diff
@@ -34,14 +28,14 @@ const NotificationPanel: React.FC<NotificationPanelProps> = ({
   if (!open) return null;
 
   return (
-    <div
-      ref={panelRef}
-      className='absolute top-[27px] right-116 right-[-116px] z-50 flex h-326 w-327 flex-col rounded-[10px] bg-white shadow-lg sm:right-0 sm:w-231'
-    >
-      <div className='flex items-center justify-between border-b-[1px] border-gray-100 px-20 py-16'>
+    <div className='absolute top-[27px] right-[-116px] z-50 flex h-326 w-327 flex-col rounded-[10px] bg-white shadow-lg sm:right-0 sm:w-231'>
```

Comment:
```suggestion
    <div className='absolute top-27 -right-116 z-50 flex h-326 w-327 flex-col rounded-[10px] bg-white shadow-lg sm:right-0 sm:w-231'>
```
❗ 이렇게 쓸 수 있어요~ 그리고 sm은 사용되지 않는 breakpoint입니다!